indra/assemblers/pybel/assembler.py

```diff
@@ -400,7 +400,7 @@ def get_ns(n):
         rel = edge_data.get('relation')
         pos_edge = \
             (u, v, ('sign', 0)) + \
-            tuple((k, (tuple(v) if isinstance(v, list) else v))
+            tuple((k, tuple(v))
```

Comment:
We should assume that all `v` from this annotations dictionary are now of type `Mapping[str,bool]` where the keys are the actual values of the annotation, and the "values" are booleans saying whether they're true or not. Just assume always true.